src/main/kotlin/uk/gov/justice/digital/hmpps/templatepackagename/ExampleResource.kt

```diff
@@ -0,0 +1,17 @@
+package uk.gov.justice.digital.hmpps.templatepackagename
+
+import org.springframework.security.access.prepost.PreAuthorize
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import java.time.LocalDateTime
+
+// TODO This controller exists to support the HMPPS Typescript template and should be removed by the bootstrap process
+@RestController
+@PreAuthorize("hasRole('ROLE_TEMPLATE_KOTLIN__UI')")
+@RequestMapping(value = ["/example"], produces = ["application/json"])
```

Comment:
shorter syntax probably better
```suggestion
@RequestMapping("/example", produces = ["application/json"])
```

Reply:
also maybe better to use `MediaType.APPLICATION_JSON_VALUE` instead?